### DISCUSSION TITLE
One class for height and width like hw or size?

### DISCUSSION THREAD

**Discussion by DasdWaller** [tailwindlabs/tailwindcss#5937] [Ideas]
Some times need to same size to width and height. For more faster way to use something like wh or hw or size or what ever...

**Comment by dance2die** [comment]
I do like the idea of having "wh/hw" to set both width and height.

In the mean time, you can create a simple plugin for utilities like following to accomplish it.  
**Tailwind CSS Play** on https://play.tailwindcss.com/4u5cIzwt9J?layout=horizontal

You can change `cssPrefix` value "to use something like wh or hw or size or what ever..."

All values available for Tailwind CSS's width utilities are available (retrieved with `theme("width")`).

```js
function square({ addUtilities, e, theme }) {
  const widthMap = theme("width");
  const cssPrefix = "sq";

  const utilities = Object.keys(widthMap).map((key) => {
    const value = widthMap[key];

    return {
      [`.${e(`${cssPrefix}-${key}`)}`]: {
        width: value,
        height: value,
      },
    };
  });

  addUtilities(utilities);
}
```

### Demo

Here is the working demo showing the rendered CSS and smart completion shown.

![Q3V05gye5Y](https://user-images.githubusercontent.com/8465237/139523656-5cf11e4c-eae7-41bf-a9c8-ae4fe5f59d52.gif)
